test/Scope/Positive.hs

```diff
@@ -226,5 +226,9 @@ tests =
     PosTest
       "Pragmas"
       $(mkRelDir ".")
-      $(mkRelFile "Pragmas.juvix")
+      $(mkRelFile "Pragmas.juvix"),
+    PosTest
```

Comment:
suggestion: Add a test that covers `using` and `hiding` syntax (unless we already have one).

Reply:
> suggestion: raise an issue in https://github.com/anoma/juvix-docs so we remember to update the language reference

done (https://github.com/anoma/juvix-docs/issues/15)